lib/tasks/import_datastream.rake

```diff
@@ -29,9 +29,8 @@ namespace :ssg do
     # DATASTREAM_FILENAMES from openscap_parser's ssg:sync
     begin
       Rails.cache.delete('latest_supported_benchmarks')
-      ENV['DATASTREAMS'] = ::Xccdf::Benchmark::
-        LATEST_SUPPORTED_VERSIONS.map do |ref_id, version|
```

Comment:
Could we get rid of this constant altogether and just use the latest version per major OS in the config?

Reply:
Good question, but a separate concern. Captured in RHICOMPL-1121.